ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -251,7 +253,7 @@ extern "C" {
                                           classList: *mut *mut *mut nsIAtom)
      -> u32;
     pub fn Gecko_GetServoDeclarationBlock(element: *mut RawGeckoElement)
-     -> *mut ServoDeclarationBlock;
+     -> ServoDeclarationBlockBorrowed;
```

Comment:
We can, but we can't get bindgen to generate that.

It's unsafe, so it's okay for now. If possible leave a scary doc comment on the gecko side. 

It's temporary since at some point *mut RawGeckoElement will become RawGeckoElementBorrowed or something

Though I'm not sure how this compiles.